deployment/modules/monitoring/main.tf

```diff
@@ -42,6 +42,10 @@ resource "google_project_service" "monitoring_api" {
 locals {
   distributor_service = "distributor-service-${var.env}"
   duration            = "5m"
+  # Calculate the threshold for majority.
+  # For odd numbers of devices, ceil(N/2) is fine, but if N is even we need to detect that and add 1:
+  majority         = ceil(var.num_expected_devices / 2) + 1 - (ceil(var.num_expected_devices / 2) - floor(var.num_expected_devices / 2))
```

Comment:
Would `ceil(1 + var.num_expected_devices / 2)` be equivalent to all of this?

Reply:
Yeah that's what I meant. Spent too long doubting myself and then being convinced I was write to remember BODMAS apparently.